docs/common/common_examples.rst

```diff
@@ -368,17 +367,16 @@ EN 1.4. Entity Configuration Response Trust Anchor
      },
      "metadata":{
          "federation_entity": {
+             "organization_name": "example TA"
              "contacts":[
                  "spid.tech@agid.gov.it"
              ],
+             "homepage_uri": "https://registry.agid.gov.it/",
+             "logo_uri":"https://registry.agid.gov.it/static/svg/logo.svg",
              "federation_fetch_endpoint": "https://registry.agid.gov.it/fetch/",
              "federation_resolve_endpoint": "https://registry.agid.gov.it/resolve/",
              "federation_list_endpoint": "https://registry.agid.gov.it/list/",
-             "homepage_uri": "https://registry.agid.gov.it/",
-             "name": "example TA"
-         },
-          "trust_mark_issuer": {
-             "federation_status_endpoint": "https://registry.agid.gov.it/trust_mark_status/"
+             "federation_trust_mark_status": "https://registry.agid.gov.it/trust_mark_status/"
```

Comment:
federation_trust_mark_status_endpoint

here in the standard
https://bitbucket.org/openid/connect/src/c0d102b05bfb4751744b6099f3f6c1a306e5d69f/openid-connect-federation-1_0.xml#lines-1247

Reply:
fixed